Makefile

```diff
@@ -3,6 +3,7 @@
 iso:
 	# Compile the bootloader
 	nasm -f elf src/bootloader.asm -o bootloader.o
+	nasm -f elf src/io/io.asm -o io.o
```

Comment:
Since we are compiling more than just the bootloader as an `asm` file now, can we modify the comment above to say `Compile assembly-level code` or something of the like?

Reply:
Done.